anyway/parsers/cbs/s3/base.py

```diff
@@ -4,11 +4,11 @@
 
 
 class S3DataClass:
-    def __init__(self):
+    def __init__(self, s3_bucket_name=None):
         self._aws_access_key = secrets.get("AWS_ACCESS_KEY")
         self._aws_secret_key = secrets.get("AWS_SECRET_KEY")
         self._s3_resource = None
-        self._s3_bucket = None
+        self._s3_bucket = self.s3_resource.Bucket(s3_bucket_name) if s3_bucket_name else None
```

Comment:
If `s3_bucket_name` evaluates to `True`, then this line will try to access an attribute of `None`?